code/game/machinery/_machinery.dm

```diff
@@ -365,6 +365,9 @@
 		if(temp_area)
 			var/obj/machinery/power/apc/temp_apc = temp_area.get_apc()
 
+			if(temp_apc && temp_apc.terminal && temp_apc.terminal.powernet)
+				temp_apc.toggle_breaker()
```

Comment:
DRY-Principle, why have two if-statements with the same conditions directly following one another? just move this into the other if